src/Microsoft.Health.Fhir.Liquid.Converter/Models/TraceInfo.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Health.Fhir.Liquid.Converter.Models
 {
     /// <summary>
     /// TraceInfo records processing details during conversion.
-    /// It is going to be implemented in coming releases.
+    /// For HL7 v2 conversion, please use its inherited class Hl7v2TraceInfo.
```

Comment:
also consider ccda case if any shared field or method.

Reply:
Yes, would it be better if we simplify the output to "UnusedLines" and "UnusedIntervals" and make them shared for both HL7 v2 and CCDA?